assets/app/views/modals/delete-resource.html

```diff
@@ -4,6 +4,21 @@
       '<strong>{{displayName ? displayName : resourceName}}</strong>'?</h1>
     <p>This<span ng-if="isProject"> will <strong>delete all resources</strong> associated with the project {{displayName ? displayName : resourceName}} and</span> <strong>cannot be undone</strong>.  Make sure this is something you really want to do!</p>
 
+    <div ng-show="typeNameToConfirm">
+      <p>Type the name of the {{typeDisplayName || (kind | humanizeKind)}} to confirm.</p>
```

Comment:
ah yep, got it

On Wed, Apr 20, 2016 at 9:55 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/modals/delete-resource.html
> <https://github.com/jwforres/origin/pull/59#discussion_r60410983>:
>
> > @@ -4,6 +4,21 @@
> >        '<strong>{{displayName ? displayName : resourceName}}</strong>'?</h1>
> >      <p>This<span ng-if="isProject"> will <strong>delete all resources</strong> associated with the project {{displayName ? displayName : resourceName}} and</span> <strong>cannot be undone</strong>.  Make sure this is something you really want to do!</p>
> >
> > +    <div ng-show="typeNameToConfirm">
> > +      <p>Type the name of the {{typeDisplayName || (kind | humanizeKind)}} to confirm.</p>
>
> We need typeDisplayName for Deployment vs Replication Controller.
>
> I'll remove the type display name from project since I opened another PR
> to add the option to the filter.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jwforres/origin/pull/59/files/c5583fe1e4738443dc6ef5e76153301cc29e1d52#r60410983>
>